irc/prometheus.go

```diff
@@ -0,0 +1,32 @@
+package irc
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	command_received = prometheus.NewCounterVec(
```

Comment:
Other metrics that I think would be interesting:

- Number of active connections
- Number of channels with non-zero users

Reply:
I added those to this PR. Should be enough examples of metrics that future things thought of can be easily added via additional PRs.